.github/workflows/nx-integration-tests.yml

```diff
@@ -0,0 +1,186 @@
+          ref: ${{ inputs.pr_head }}
+          pr_base: ${{ inputs.pr_base }}
+      
+  integration-tests:
+    needs: 
+      - matrix-calculator
+      - nx-project-setup
+    strategy:
+      fail-fast: false
+      matrix:
+        php_version: ${{ fromJSON(needs.matrix-calculator.outputs.php_versions) }}
+        database_version: ${{ fromJSON(needs.matrix-calculator.outputs.database_versions) }}
+        search_version: ${{ fromJSON(needs.matrix-calculator.outputs.search_versions) }}
+        message_queue_version: ${{ fromJSON(needs.matrix-calculator.outputs.message_queue_versions) }}
+        cache_version: ${{ fromJSON(needs.matrix-calculator.outputs.cache_versions) }}
+        http_cache_version: ${{ fromJSON(needs.matrix-calculator.outputs.http_cache_versions) }}
+    runs-on: ubuntu-latest
+    steps:
+      # - name: Running With
+      #   shell: bash
+      #   run: |
+      #     echo "PHP Versions: ${{ matrix.php_version }}"
+      #     echo "database Versions: ${{ matrix.database_version }}"
+      #     echo "search Versions: ${{ matrix.search_version }}"
+      #     echo "message_queue Versions: ${{ matrix.message_queue_version }}"
+      #     echo "cache Versions: ${{ matrix.cache_version }}"
+      #     echo "http_cache Versions: ${{ matrix.http_cache_version }}"
+
+      - name: Project Cache
+        uses: actions/cache/restore@v3
+        with:
+          path: main
+          key: ${{ runner.os }}-project-${{ inputs.ref }}
+      
+      # could probably swap this to a docker container in future
+      - name: Install NX
+        working-directory: ./main
+        run: |
+          npm install -g nx@15.4.1
+
+      - name: Print Affected
+        working-directory: ./main
+        run: |
+          AFFECTED_OUTPUT=/tmp/affect.json
+          nx print-affected --head=HEAD --base=remotes/origin/${{ inputs.pr_base }} > ${AFFECTED_OUTPUT}
+          cat ${AFFECTED_OUTPUT}
+          echo "Affected Projects: $(jq .projects ${AFFECTED_OUTPUT})"
+
+      - name: Setup Warden Environment
+        uses: mage-os/github-actions/warden/setup-environment@main
+        with:
+          php_version: ${{ matrix.php_version }}
+          database: ${{ matrix.database_version }}
+          search: ${{ matrix.search_version }}
+          rabbitmq: ${{ matrix.message_queue_version }}
+          redis: ${{ matrix.cache_version }}
+          varnish: ${{ matrix.http_cache_version }}
+          base_directory: "./main"
+
+      - name: Setup config for Integration tests
+        uses: mage-os/github-actions/warden/integration-tests@main
+        with:
+          search: ${{ matrix.search_version }}
+          rabbitmq: ${{ matrix.message_queue_version }}
+          redis: ${{ matrix.cache_version }}
+          run_memory_test: 0
+          run_magento_integration_tests: 0
+          run_magento_integration_tests_real_suite: 0
+          base_directory: "./main"
+
+      # - name: Run Memory Tests

```

Comment:
Do we need this? If not, can we remove it?

Reply:
sure, have removed